### ISSUE TITLE
Add parallel transport to SPD with BW metric

### ISSUE THREAD

**Issue by mateuszbaran** [JuliaManifolds/Manifolds.jl#480]
Described here: https://arxiv.org/abs/2109.05768 .

**Comment by mateuszbaran** [comment]
I think $d_i, \delta_i$ are eigenvalues of SPD matrices, that's how it's used in Section 2. $d$ should correspond to $D$ and $\delta$ to $\Delta$.

**Comment by mateuszbaran** [comment]
Regarding ODE solving, I think there are three reasonable directions:
1. Use ManifoldDiffEq.jl to solve the ODE.
2. Work in charts (not polished yet).
3. Roll out a simple first or second order solver for a particular case.

**Comment by kellertuer** [comment]
> I think di,δi are eigenvalues of SPD matrices, that's how it's used in Section 2. `d` should correspond to `D` and `\delta` to Δ.

but in the formula they use (delta_j+delta_i) / (d_i+d_j) so if _both_ are eigenvalues of some SPD matrices, I do not know which one wis which. I also have not seen the single-index delta in sec 2?

**Comment by mateuszbaran** [comment]
This suggests how `d_i` and `D` are related:
![image](https://user-images.githubusercontent.com/2551062/177629141-c739614e-cbbd-40bc-8ed0-62e340bdb3b2.png)
so here:
![image](https://user-images.githubusercontent.com/2551062/177629397-557bce70-af71-4029-b10a-212cb96fe549.png)
I think `d_i` are eigenvalues in `D` and `\delta_i` are eigenvalues in `\Delta`.
But they could have written that more clearly.

**Comment by mateuszbaran** [comment]
> Ok, neither of those three is what I could provide, since I also do not seem able to get their notation, this might not be an issue for me.

Yes, none of them is particularly simple. The easiest thing to start here is most likely a rolled-out implementation of a solver for the Sasaki metric I still mean to finish...

**Comment by kellertuer** [comment]
> I think `d_i` are eigenvalues in `D` and `\delta_i` are eigenvalues in `\Delta`.
> But they could have written that more clearly.

That is possible, for D it is impicitly mentioned somewhere in section 2, for Delta that is not mentioned at all. Still it only solves a specific case, so this Issue really has to wait until we have some ODE framework.
